Dockerfile

```
@@ -30,7 +30,9 @@ COPY --from=builder /workspace/manager .
 COPY config/recipes/ /opt/sro/recipes/
 COPY manifests /manifests
 
-USER nonroot:nonroot
+RUN useradd -r -u 499 nonroot
+RUN groupadd   -g 499 nonroot
```

This line causes `make local-image-build` to fail for me, because "groupadd: group 'nonroot' already exists" from the useradd command.

```suggestion
RUN groupadd   -g 499 nonroot
```